Add video-to-video task

MrShahzebKhoso opened the pull request:
- Added about.md with a definition and summary of the task.
- Added data.ts with task metadata:
    -   Datasets (VIRESET, LongV-EVAL, SeedVR_VideoDemos)
    -   Models (e.g., Lucy-Edit-Dev, SeedVR2, Shape-for-Motion, etc.)
    -   Demo Spaces (lucy-edit-dev, SeedVR2-3B)
    -   Task summary describing scope and applications.

sergiopaniego commented on packages/tasks/src/tasks/video-to-video/about.md:
```diff
@@ -0,0 +1,190 @@
+- [Shape-for-Motion](https://huggingface.co/papers/2506.22432) - A video editing framework that uses time-consistent 3D meshes and dual-propagation to apply precise single-frame edits across entire videos.
+- [Generative Video Propagation](https://arxiv.org/html/2412.19761v1) - A framework that propagates first-frame edits across videos, enabling insertion, removal, background replacement, and multi-edit consistency.
+
+### Codes
```
```suggestion
### Repositories 
```

sergiopaniego reviewed (commented):
Thanks for the contribution! I agree with Merve's suggestions and just leaving a few additional minor nits :)

MrShahzebKhoso commented on packages/tasks/src/tasks/video-to-video/data.ts:
```diff
@@ -0,0 +1,66 @@
+		},
+	],
+	demo: {
+		inputs: [],
```
Submitted the PR (#10) there. It includes two gif files (input and output). The demo is of Video Super-Resolution so the input is low resolution video while output is high resolution.

https://huggingface.co/datasets/huggingfacejs/tasks/discussions/10

MrShahzebKhoso commented:
Thank you for considering this PR and taking time to review. 

Updated both the about.md and data.ts files based on all the helpful feedback from @merveenoyan and @sergiopaniego — thank you both for the detailed and thoughtful reviews. Really appreciate the guidance.

Also added a Video Super-Resolution demo (low-res to high-res) via PR [#10](https://huggingface.co/datasets/huggingfacejs/tasks/discussions/10)
 on the datasets repo.

Everything is now cleaned up, clarified, and aligned with the review suggestions.

Vaibhavs10 reviewed (commented):
make sure that the changes reflect here: as well `packages/tasks/src/tasks/index.ts`

cc: @merveenoyan

MrShahzebKhoso commented:
Thanks for catching that, @Vaibhavs10!
I’ve added the corresponding entry to packages/tasks/src/tasks/index.ts so the changes are now reflected there as well.
Appreciate the review and guidance!

pcuenca commented on packages/tasks/src/tasks/video-to-video/data.ts:
```diff
@@ -0,0 +1,68 @@
+	models: [
+		{
+			description:
+				"Model for editing outfits, character, and scenery in videos.",
```
```suggestion
			description: "Model for editing outfits, character, and scenery in videos.",
```
